salt/fileserver/__init__.py

```diff
@@ -505,16 +505,6 @@ def find_file(self, path, saltenv, back=None):
                         continue
                     args = comp.split('=', 1)
                     kwargs[args[0]] = args[1]
-        if 'env' in kwargs:
-            salt.utils.warn_until(
-                'Boron',
-                'Passing a salt environment should be done using \'saltenv\' '
-                'not \'env\'. This functionality will be removed in Salt '
-                'Boron.'
-            )
-            saltenv = kwargs.pop('env')
-        elif 'saltenv' in kwargs:
```

Comment:
This is also removed because it is not sensible to allow `saltenv` to be passed twice into this function.